universal-application-tool-0.0.1/app/services/program/predicate/PredicateDefinition.java

```diff
@@ -10,6 +10,7 @@
 import com.google.common.collect.ImmutableSet;
 import services.question.types.QuestionDefinition;
 
+/** Visibility configuration of a program screen (block). */
```

Comment:
suggestion (blocking): predicates technically could be used for other purposes, such as eligibility (in the future). Recommend generalizing this comment
```suggestion
/** Conditional logic associated with a single program screen (block). This may be used to (for example) show or hide a block based on the answers an applicant provides. */
```

Reply:
Great suggestions! Thanks for the input.